Apps/RaptureIntTests/src/test/java/rapture/lock/LockWorkflowTest.java

```diff
@@ -562,7 +563,9 @@ public void testBlockingLockWithinWorkflows() {
 		       	lockHandle.setLockName(decisionApi.getContextValue(orderUri+"#0", "lockName"));	
 	       	}
         }
-        Assert.assertTrue(lockApi.releaseLock(lockUri.toString(), lockConfig.getName(), lockHandle));
+        try {
```

Comment:
if the
1.  behavior is documented in the api doc
2. the exception message states "you aren't the owner of the lock.."

 then no need to open a new ticket.